.travis.yml

```diff
@@ -0,0 +1,94 @@
+# Copyright (c) 2020 Arm Limited. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the License); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: sh
+os: linux
+dist: xenial
+
+cache:
+  pip: true
+  directories:
+    - ${HOME}/.cache/deps
+
+addons:
+  apt:
+    packages:
+    - ninja-build
+
+matrix:
+  include:
+
+    - &cmake-build-test
+      stage: "CMake"
+      name: "CMake blinky example - develop (K64F)"
+      env: NAME=cmake_test EXAMPLE_NAME=mbed-os-example-blinky TARGET_NAME=K64F PROFILE=develop
+      language: python
+      python: 3.8
+      install:
+      # Install arm-none-eabi-gcc
+        - pushd /home/travis/build && mkdir arm-gcc && cd arm-gcc
+        - curl -L0 "https://developer.arm.com/-/media/Files/downloads/gnu-rm/9-2020q2/gcc-arm-none-eabi-9-2020-q2-update-x86_64-linux.tar.bz2?revision=05382cca-1721-44e1-ae19-1e7c3dc96118&la=en&hash=D7C9D18FCA2DD9F894FD9F3C3DC9228498FA281A" --output gcc-arm-none-eabi-9-2020-q2-update.tar.bz2
+        - tar xf gcc-arm-none-eabi-9-2020-q2-update.tar.bz2
+        - export PATH="${PWD}/gcc-arm-none-eabi-9-2020-q2-update/bin:$PATH"
+        - popd
+        - arm-none-eabi-gcc --version
+        - pip install --upgrade cmake
+        # Install the changes added in the pull request
+        - pip install git+https://github.com/ARMmbed/mbed-tools.git@refs/pull/${TRAVIS_PULL_REQUEST}/head
```

Comment:
Would `cd ${TRAVIS_BUILD_DIR} && pip install .` not work instead of fetching again from github?

Reply:
In order to do that, the contents of the PR needs to be fetched. Directly installing from Github seems simpler.

Reply:
Travis doesn't automatically fetch and check out your PR? Seems strange to me. I need to read some more about how Travis works then.

Reply:
It turns out that Travis will fetch the contents of the PR. So changing the install command to `pip install .`.